Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG BASE_IMAGE=northamerica-northeast2-docker.pkg.dev/kontinue/library/ubuntu:jammy
+ARG GOLANG_IMAGE=northamerica-northeast2-docker.pkg.dev/kontinue/library/golang:1.19
+
+FROM ${BASE_IMAGE} AS base
+
+RUN set -x && \
+	apt-get update && \
+	apt-get upgrade -y && \
+	apt-get install  -y \
+	jq curl git ca-certificates gnutls-bin && \
```

Comment:
Are all of these packages required? 

I'm not sure that we need gnutls-bin and ca-certificates for the build. And I expected curl to be in the base image.

Can we install pinned versions of git and jq through curl like we do ytt?